example/apps/test_security/tests/__init__.py

```diff
@@ -499,25 +501,72 @@ def test_output_logged_request_should_be_related_with_object_selected_in_decorat
 
         responses.add(responses.GET, 'http://test.cz', body='test')
         assert_equal(get_all_request_related_objects(), [])
-        with log_request_related_objects([user1]):
+        with log_request_related_objects(slug='test1', related_objects=[user1]):
             assert_equal(set(get_all_request_related_objects()), {user1})
 
             requests.get('http://test.cz')
             assert_equal(OutputLoggedRequest.objects.count(), 1)
-            assert_equal(OutputLoggedRequest.objects.get().related_objects.get().content_object, user1)
-            with log_request_related_objects([user2]):
+            assert_equal(OutputLoggedRequest.objects.get().related_objects.get().object, user1)
+            assert_equal(OutputLoggedRequest.objects.first().slug, 'test1')
+            with log_request_related_objects(slug='test2',related_objects=[user2]):
                 assert_equal(set(get_all_request_related_objects()), {user1, user2})
                 requests.get('http://test.cz')
                 assert_equal(OutputLoggedRequest.objects.count(), 2)
                 assert_equal(
                     {
-                        related_object.content_object
+                        related_object.object
                         for related_object in OutputLoggedRequest.objects.first().related_objects.all()
                     },
                     {user1, user2}
                 )
+                assert_equal(OutputLoggedRequest.objects.first().slug, 'test2')
             assert_equal(set(get_all_request_related_objects()), {user1})
             requests.get('http://test.cz')
             assert_equal(OutputLoggedRequest.objects.count(), 3)
-            assert_equal(OutputLoggedRequest.objects.first().related_objects.get().content_object, user1)
+            assert_equal(OutputLoggedRequest.objects.first().related_objects.get().object, user1)
+            assert_equal(OutputLoggedRequest.objects.first().slug, 'test1')
         assert_equal(get_all_request_related_objects(), [])
+        assert_is_none(get_request_slug())
+
+    @data_provider('create_user')
+    def test_task_log_is_processing_should_return_if_task_is_active_or_waiting(self, user):
+        assert_false(sum_task.is_processing())
+
+        celery_task_log = CeleryTaskLog.objects.create(
+            celery_task_id='test',
+            name='sum_task',
+            stale=now(),
+            estimated_time_of_first_arrival=now()
+        )
+        celery_task_log.related_objects.add(user)
+
+        assert_true(sum_task.is_processing())
+        assert_true(sum_task.is_processing(related_objects=(user,)))
```

Comment:
[user] instead of (user,), there are more examples, I think we settled on the list representation for a single element array